compiler/frontend/src/org/jetbrains/kotlin/psi/psiUtil/ktPsiUtil.kt

```diff
@@ -454,3 +454,17 @@ fun checkReservedPrefixWord(sink: DiagnosticSink, element: PsiElement, word: Str
     }
 }
 
+fun KtNameReferenceExpression.getPrimaryConstructorParameterWithSameName(): KtParameter? {
```

Comment:
I think this is far too specific to live in a general PSI utlilities class; I'd move it into the quickfix file.